lib/payment_calculator/ecf/contract/output_payment_calculations.rb

```diff
@@ -19,7 +19,7 @@ def output_payment_per_participant(band)
         end
 
         def output_payment_for_event(event_type:, total_participants:, band:)
-          total_participants * output_payment_per_participant_for_event(event_type: event_type, band: band)
+          [band.number_of_participants_in_this_band(total_participants), total_participants].min * output_payment_per_participant_for_event(event_type: event_type, band: band)
```

Comment:
Can we add a unit test in `spec/lib/payment_calculator/ecf/contract/output_payment_calculations_spec.rb` that will prevent anyone reintroducing this bug?

Reply:
Ok, I will add it, although it's covered by the retention table feature test

Reply:
@plisovin Did that test fail?

Reply:
yeah, and I need to fix the way this feature test is written as it's using Double rather than expected method

Reply:
Added in c501e66